presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeAggregations.java

```diff
@@ -135,6 +135,29 @@ public void testMixedDistinctAggregations()
         assertQuery(session, "SELECT max(orderstatus), COUNT(orderkey), sum(DISTINCT orderkey) FROM orders");
     }
 
+    @Test
+    public void testEmptyGroupingSets()
+    {
+        // This should return a single row with the global aggregation.
```

Comment:
nit: "This short return" -> "Returns a single..."

ditto other places

Reply:
Done.